system-addon/content-src/message-center/components/SnippetBase.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+const defaultStyles = {
+  container: {
+    position: "fixed",
+    bottom: 0,
+    left: 0,
+    right: 0,
+    padding: "20px",
+    backgroundColor: "white",
+    fontSize: "12px",
+    lineHeight: "16px",
+    boxShadow: "0 -1px 4px 0 rgba(12, 12, 13, 0.1)"
+  },
+  innerWrapper: {
+    maxWidth: "992px",
+    margin: "0 auto",
+    display: "flex",
+    alignItems: "center"
+  },
+  blockButton: {
+    background: "none",
+    border: 0,
+    display: "block",
+    position: "absolute",
+    top: "10px",
+    right: "10px",
+    height: "16px",
+    width: "16px",
+    backgroundImage: "url(resource://activity-stream/data/content/assets/glyph-dismiss-16.svg)",
+    opacity: 0.5
+  }
+};
+
+export class SnippetBase extends React.PureComponent {
+  render() {
+    const {props} = this;
+
+    // Extend default styles
+    const styles = Object.assign({}, defaultStyles);
+    if (props.styles) {
+      for (const key in props.styles) {
+        if (Object.prototype.hasOwnProperty.call(props.styles, key)) {
+          styles[key] = Object.assign({}, styles[key], props.styles[key]);
+        }
+      }
+    }
+
+    // Don't render anything if no ID is present
+    if (!props.id) {
+      return null;
```

Comment:
Should this be at the top of the `render` method?

Reply:
I should actually remove this, I added a guard earlier on than this template